dialog/file.go

```diff
@@ -946,6 +946,10 @@ func getFavoriteOrder() []string {
 }
 
 func hasAppFiles(a fyne.App) bool {
+	if a.UniqueID() == "testApp" {
```

Comment:
This kind of pattern always seems fragile to me and something we should try to avoid.. but if we can't, let's make the magic string something less likely to collide with user code, maybe even just "io.fyne.testApp". I could easily see someone testing out fyne and initializing with `app.NewWithID("testApp")`

Reply:
I wanted to do a type check but the test types are private. The magic string has been set a long time ago, there may be impact to changing it now...

Reply:
Can the magic string not be exported as a const somewhere?